projects/common-components/src/public-api.ts

```diff
@@ -1,8 +1,4 @@
-/*
- * Public API Surface of common-components
- */
-
 export * from "./lib/common-components.service";
 export * from "./lib/common-components.component";
-export * from "./lib/button/button.component";
 export * from "./lib/common-components.module";
+export * from "./lib/button/button.component";
```

Comment:
What is your goal by making these changes to the project?

Reply:
In this file button component was in the middle of common-components, i just changed their place, at first all common-components files, the next is button component

Reply:
I remain it because got this time i worked with this folder))Done